readme.md

```diff
@@ -12,3 +12,23 @@ pnpm install
 # setup git hooks
 npx simple-git-hooks
 ```
+
+## Deployments 
+
+### Access API
+
+There's 3 environments prodution, staging and dev. The URLs are:
+- Prodution: `https://access-api.web3.storage`
+- Staging: `https://access-api-staging.web3.storage`
+- Dev: `https://w3access-dev.protocol-labs.workers.dev`
```

Comment:
```suggestion
- Prodution: https://access-api.web3.storage
- Staging: https://access-api-staging.web3.storage
- Dev: https://w3access-dev.protocol-labs.workers.dev
```

Unless there is a good reason to make these links non clickable, I'd suggest making them clickable